pkg/ccl/changefeedccl/changefeed_dist_test.go

```diff
@@ -480,6 +481,83 @@ func (rdt *rangeDistributionTester) balancedDistributionUpperBound(numNodes int)
 	return int(math.Ceil((1 + rebalanceThreshold.Get(&rdt.lastNode.ClusterSettings().SV) + 0.1) * 64 / float64(numNodes)))
 }
 
+// Additional helper function
+func anyGreaterThanThirty(counts []int) bool {
+	for _, count := range counts {
+		if count > 30 {
+			return true
+		}
+	}
+	return false
+}
+
+type strategyOpt struct {
+	set bool
```

Comment:
nit: technically this bool isn't necessary since you could check if the string is empty